### ISSUE TITLE
aws_autoscaling_attachment: incorrect parameter name "alb_target_group_arn"

### ISSUE THREAD

**Issue by steeef** [hashicorp/terraform-provider-aws#5407]
### Terraform Version

```
❯ terraform -v
Terraform v0.11.7
+ provider.aws v1.29.0
```

### Affected Resource(s)

* aws_autoscaling_attachment

### Terraform Configuration Files

```hcl

resource "aws_autoscaling_attachment" "elasticsearch" {
  autoscaling_group_name = "${aws_autoscaling_group.elasticsearch.id}"
  alb_target_group_arn   = "${var.target_group}"
}

```

### Expected Behavior

When attaching an NLB target group to an autoscaling group, I expected to find a parameter named `lb_target_group_arn`.

### Actual Behavior

Parameter was named `alb_target_group_arn`:
https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_autoscaling_attachment.go#L33
Additionally, docs refer to an ALB target group:
https://github.com/terraform-providers/terraform-provider-aws/blob/master/website/docs/r/autoscaling_attachment.html.markdown

This seems to suggest that only an ALB target group can be attached to an autoscaling group. However it's possible to attach NLB target groups as well.

**Comment by jowrjowr** [comment]
This is still an issue. Came here to report this.

The actual label itself isn't too big of a deal but the documentation is the real nuisance. If this is documented in https://www.terraform.io/docs/providers/aws/r/autoscaling_attachment.html to specifically note it works for both ALB and NLB ARNs, it'd really help.

**Comment by MarcClusterman** [comment]
This continues to cause confusion, it would be nice for the docs to at least explain.

**Comment by jar-b** [comment]
Closed by #30828, merged to `main` via #31392
